pkg/gadgettracermanager/gadgettracermanager.go

```diff
@@ -38,6 +38,7 @@ import (
 import "C"
 
 type GadgetTracerManager struct {
+	pb.UnimplementedGadgetTracerManagerServer
```

Comment:
Is it necessary to add this here? It seems to provide default implementation that `GadgetTracerManager` will override anyway. I'm not saying to remove it, just trying to understand.

Reply:
AFAIU, to register services using the methods generated by `protoc-gen-go-grpc`, we **must** do this or disable this requirement by using option require_unimplemented_servers without modifying the code. However, the second option is not recommended: https://github.com/grpc/grpc-go/blob/master/cmd/protoc-gen-go-grpc/README.md#future-proofing-services